Trimble.Modus.Components/Controls/TMDataGrid/ColumnTypes/BooleanColumn.xaml.cs

```diff
@@ -0,0 +1,16 @@
+namespace Trimble.Modus.Components;
+
+public partial class BooleanColumn
```

Comment:
Add the parent class DataGridColumn. Easy to see that it is a subclass without looking at the xaml file
`public partial class BooleanColumn : DataGridColumn`

Reply:
Done